### PR TITLE
RAC-147: add new parameters to product and product model exports

### PR DESCRIPTION
**Pull request by juliensnz**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Pim/Enrichment/Component/Product/Connector/Job/JobParameters/ConstraintCollectionProvider/ProductXlsxExport.php [hunk 1]
```diff
@@ -44,6 +45,19 @@ public function getConstraintCollection()
...
+                'groups' => ['Default', 'FileConfiguration'],
+            ]
+        );
+        $constraintFields['header_with_label'] = new Type(
+            [
+                'type'   => 'bool',
+                'groups' => ['Default', 'FileConfiguration'],
+            ]
+        );
+        $constraintFields['file_locale'] = new ActivatedLocale(['groups' => ['Default', 'FileConfiguration']]);
```
Comment (tseho) [inline src/Akeneo/Pim/Enrichment/Component/Product/Connector/Job/JobParameters/ConstraintCollectionProvider/ProductXlsxExport.php:60]: is the `file_locale` required if `with_label` is set to true ?

Comment (tseho) [reply src/Akeneo/Pim/Enrichment/Component/Product/Connector/Job/JobParameters/ConstraintCollectionProvider/ProductXlsxExport.php:60]: If that's the case, we should add a conditional validation, no ?

Comment (juliensnz) [reply src/Akeneo/Pim/Enrichment/Component/Product/Connector/Job/JobParameters/ConstraintCollectionProvider/ProductXlsxExport.php:60]: yes, you are right

Comment (tseho) [reply src/Akeneo/Pim/Enrichment/Component/Product/Connector/Job/JobParameters/ConstraintCollectionProvider/ProductXlsxExport.php:60]: you can validate it in the back with a callback validator on `$baseConstraint`

**Pull request merged**